tests/test_pandaspysparklivyclient.py

```diff
@@ -102,8 +116,21 @@ def test_execute_sql_pandas_pyspark_livy_some_exception():
     execute_m.return_value = (False, some_exception)
 
     try:
-        result = client.execute_sql(command)
+        client.execute_sql(command)
         assert False
     except DataFrameParseException as e:
         execute_m.assert_called_with('sqlContext.sql("""{}""").toJSON().take({})'.format(command, 10))
         assert e.out == some_exception
+
+@with_setup(_setup, _teardown)
+def test_execute_sql_pandas_pyspark_extract_strings():
+    for (s, a) in [("[]", []),
+                   ("  ['afz']  ", ["afz"]),
+                   ("['zzz', 'aaa', 'fff']", ['zzz', 'aaa', 'fff']),
+                   (" ['aa\\'a',\n'q\\'']", ["aa'a", "q'"]),
+                   ("['']", ['']),
```

Comment:
I'm just tesing when user strings contain new lines. I'd also like to see how it handles other escaped characters.